src/power_grid_model_io/converters/vision_excel_converter.py

```diff
@@ -37,16 +37,21 @@ def __init__(
         self,
         source_file: Optional[Union[Path, str]] = None,
         language: str = "en",
+        terms_changed: Optional[dict] = None,
         mapping_file: Optional[Path] = None,
         log_level: int = logging.INFO,
```

Comment:
This argument came to me as well, but since they are parameters of different type, I went with the more logical order. Plus, positional parameters has its own share of problems and ideally should be discouraged in non-toy sccripts mo.

Reply:
100% agree but not everyone is smart enough 😬